datajoint_utilities/misc.py

```diff
@@ -0,0 +1,43 @@
+import datajoint as dj
+from pymysql import  OperationalError
+
+"""
+Misc helper functions to support dev. Chris Brozdowski <CBroz@datajoint.com>
```

Comment:
What could be a more descriptive module name for these functions?

Reply:
I went with `dev_helpers`. The standard label for helpers like this is `utils`, but that seemed repetitive when the whole repo is utilities. 

Thanks for your review!